src/base/tests/test_util.py

```diff
@@ -485,6 +485,27 @@ def test_explode_mult_filters(self):
             cr.execute(q)
             self.assertTrue(all(x for (x,) in cr.fetchall()))
 
+    def test_explode_query_range(self):
```

Comment:
```suggestion
    @mute_logger("odoo.upgrade.util.pg.explode_query_range"}.
    def test_explode_query_range(self):
```

Reply:
Done, also the last run with l10n_cr was green :) https://upgradeci.odoo.com/upgradeci/change/KktpRCzA6naVmtVyUHtA9Q